pages/onboarding/create.tsx

```diff
@@ -1,34 +1,26 @@
-import { WizardContext } from 'react-albus';
 import styled from 'styled-components';
-import { BaseXL, BaseM } from 'components/core/Text/Text';
+import { BaseM, BaseXL } from 'components/core/Text/Text';
 import { Button } from 'components/core/Button/Button';
-import FullPageCenteredStep from 'flows/shared/components/FullPageCenteredStep/FullPageCenteredStep';
-import { useCallback, useEffect } from 'react';
-import { useWizardCallback } from 'contexts/wizard/WizardCallbackContext';
+import FullPageCenteredStep from 'flows/../../src/components/Onboarding/FullPageCenteredStep/FullPageCenteredStep';
+import { useCallback } from 'react';
 import { useTrack } from 'contexts/analytics/AnalyticsContext';
 import { VStack } from 'components/core/Spacer/Stack';
+import { useRouter } from 'next/router';
+import { getStepUrl } from 'components/Onboarding/constants';
+import { OnboardingFooter } from 'components/Onboarding/OnboardingFooter';
 
-type ConfigProps = {
-  onNext: () => void;
-};
-
-function useWizardConfig({ onNext }: ConfigProps) {
-  const { setOnNext } = useWizardCallback();
-
-  useEffect(() => {
-    setOnNext(onNext);
-  }, [setOnNext, onNext]);
-}
-
-function CreateFirstCollection({ next }: WizardContext) {
+export default function CreateFirstCollection() {
   const track = useTrack();
 
+  const { push, query, back } = useRouter();
   const handleNextClick = useCallback(() => {
     track('Start new collection');
-    next();
-  }, [next, track]);
 
-  useWizardConfig({ onNext: handleNextClick });
+    push({
+      pathname: getStepUrl('organize-collection'),
+      query: { ...query },
```

Comment:
any reason not to just do `query: query`?

Reply:
nothing but paranoia lol